### PR TITLE
Events list

### PR DESCRIPTION
**Pull request by Pete-Fowler**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
```
Comment (JeffBaumgardt) [inline .vscode/settings.json]: You might want to update the gitignore to include the .vscode directory. This shouldn't be a shared folder/file set.

File: app/components/footer.tsx [hunk 2]
```diff
@@ -1,5 +1,7 @@
-import React from "react";
 import { Link } from "@remix-run/react";
+import styles from '../styles/index.css';
```
Comment (JeffBaumgardt) [inline app/components/footer.tsx]: When importing paths try to use absolute paths vs relative. In this case the import would be `import styles from '~/styles/index.css`

File: app/components/footer.tsx [hunk 3]
```diff
@@ -1,5 +1,7 @@
...
+
+export const links = () => [{ rel: "stylesheet", href: styles }];
```
Comment (JeffBaumgardt) [inline app/components/footer.tsx]: That said the main index of styles is imported in the root.tsx file so you shouldn't have to add them here.

File: content/events/2023-10-18.mdx [hunk 4]
```diff
@@ -1,9 +1,9 @@
 ---
 title: Future Talk
-date: 2022-10-18
+date: 2023-10-18
 description: Placeholder description
 location: 1595 Wynkoop St, Denver, CO 80202
 speakers:
-  - name: John Doe
+  - name: Bill Gates, Linus Torvalds, Snoop Dog
```
Comment (JeffBaumgardt) [inline content/events/2023-10-18.mdx:7]: Now that's a talk I would go to. Bill Gates and Linus Torvalds would be very interesting talks. And to top it off with Snoop? I would love that to be a panel discussion about the future in tech with Snoop just sitting back saying yeas to everything with a fun aniocodate.

Comment (Pete-Fowler) [reply content/events/2023-10-18.mdx:7]: Updates made - thanks. You'd be in good company presenting with titans of industry.

**Review by JeffBaumgardt — APPROVED** [review]

**Comment by JeffBaumgardt** [comment]
Closes #45

**Pull request merged**